pkg/scanners/clairify/convert.go

```diff
@@ -12,14 +12,55 @@ import (
 	v1 "github.com/stackrox/scanner/generated/scanner/api/v1"
 )
 
-func convertNodeToVulnRequest(node *storage.Node) *v1.GetNodeVulnerabilitiesRequest {
-	return &v1.GetNodeVulnerabilitiesRequest{
+func convertNodeToVulnRequest(node *storage.Node, inventory *storage.NodeInventory) *v1.GetNodeVulnerabilitiesRequest {
+	req := &v1.GetNodeVulnerabilitiesRequest{
 		OsImage:          node.GetOsImage(),
 		KernelVersion:    node.GetKernelVersion(),
 		KubeletVersion:   node.GetKubeletVersion(),
 		KubeproxyVersion: node.GetKubeProxyVersion(),
 		Runtime:          convertContainerRuntime(node.GetContainerRuntime()),
+		Components:       nil,
 	}
+	if inventory != nil && inventory.GetComponents() != nil {
+		req.Components = convertComponents(inventory.GetComponents())
+	}
+	return req
+}
+
+func convertComponents(c *storage.NodeInventory_Components) *v1.Components {
+	components := &v1.Components{
+		Namespace:          c.Namespace,
```

Comment:
nit: `c.GetNamespace()`
This is currently not absolutely necessary yet as the caller filters out the `nil` case, but would better bulletproof the code.